doc/modules/ROOT/pages/index.adoc

```diff
@@ -7,6 +7,8 @@
 
 This is a complete example presenting how to use Hazelcast as a sidecar container for your applications deployed on Kubernetes.
 
+include::{templates-url}/link-to-repo.adoc[]
```

Comment:
I haven't read the all content but couldn't see any link to repo at the first glance. Hence adding this.